### PR TITLE
Delivery for Reporter Instant App - Sprint1 - AGOL-2024.R01

### PR DESCRIPTION
**Pull request by sumitzarkar**
## Implemented following items
- Loading map
- Creating side panel
- Layer list 

## Following items are implemented and are still in testing:
- Feature List
- Feature Details (when selected feature using feature list)

## Tested on following platforms:
image

## QA Checklist:

[QAChecklist-_CrowdsourceReporter_Instant_App_-_AGOL2024_-_Sprint1.xlsx](https://github.com/Esri/solutions-components/files/13920413/QAChecklist-_CrowdsourceReporter_Instant_App_-_AGOL2024_-_Sprint1.xlsx)

## Test cases & Execution report:
[ESRILG_-TestScenarios___TestCases_-_CrowdsourceReporter_Instant_App-_AGOL2024_-_Sprint1.xlsx](https://github.com/Esri/solutions-components/files/13920409/ESRILG_-TestScenarios___TestCases_-_CrowdsourceReporter_Instant_App-_AGOL2024_-_Sprint1.xlsx)

### CODE REVIEW

File: src/components/crowdsource-reporter/crowdsource-reporter.tsx [hunk 1]
```diff
@@ -145,23 +249,45 @@ export class CrowdsourceReporter {
...
-        </slot>
+        <div>
+          {validConfiguration &&
+            <calcite-shell content-behind >
+              {this._getReporter()}
+              {this._getMapNode()}
+            </calcite-shell>
+          }
+          {!validConfiguration &&
+            <calcite-notice class="error-msg" icon="configure" kind="danger" open>
```
Comment (jmhauck) [inline src/components/crowdsource-reporter/crowdsource-reporter.tsx:285]: @sumitzarkar minor thing but for consistency across the repo, if you have more than one attribute please list them on separate lines.

File: src/demos/feature-list.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <script src="https://jsdev.arcgis.com/4.29/"></script>
+    <script type="module" src="../solutions-components.esm.js"></script>
+
+    <script>
+    </script>
+  </head>
+
+  <!-- <body class="calcite-mode-dark"> -->
+  <body>
+    <feature-list/>
```
Comment (jmhauck) [inline src/demos/feature-list.html:41]: Please make make this functional

File: src/demos/layer-list.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <script src="https://jsdev.arcgis.com/4.29/"></script>
+    <script type="module" src="../solutions-components.esm.js"></script>
+
+    <script>
+    </script>
+  </head>
+
+  <!-- <body class="calcite-mode-dark"> -->
+  <body>
+    <layer-list/>
```
Comment (jmhauck) [inline src/demos/layer-list.html:41]: Please make make this functional

**Review by jmhauck — APPROVED** [review]
Great start...nice work.

**Comment by sumitzarkar** [comment]
@jmhauck 

We have implemented above review comments, we will deilver it in the next code drop

**Pull request merged**